File: nos2jdbc-tutorial-wildfly/src/main/java/nos2jdbc/tutorial/wildfly/entity/Member.java
package nos2jdbc.tutorial.wildfly.entity;

import java.util.List;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.OneToMany;

@Entity
public class Member extends EntityBase {
    @Column
    public String name;
    
    @OneToMany(mappedBy ="member")
    public List<ClubMemberRel> clubMemberRelList;
}

File: nos2jdbc-tutorial-kotlin-spring/src/main/kotlin/nos2jdbc/tutorial/kotlinspring/entity/Member.kt
package nos2jdbc.tutorial.kotlinspring.entity

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.GeneratedValue
import jakarta.persistence.Column
import jakarta.persistence.OneToMany
@Entity
class Member : EntityBase() {
    @Column
    var name: String = ""
    
    @OneToMany(mappedBy ="member")
    var clubMemberRelList: List<ClubMemberRel> = mutableListOf()
}
/*
@Entity
data class Member(
    @Id
    @GeneratedValue
    var id: Long = 0,
    
    @Column
    var name: String = "",
    
    @OneToMany(mappedBy ="member")
    var clubMemberRelList: List<ClubMemberRel> = mutableListOf()

) {
	override fun hashCode(): Int {
		return id.hashCode();
	}
}
*/